src/vs/workbench/contrib/chat/browser/chatContentParts/chatAttachmentsContentPart.ts

```diff
@@ -39,8 +45,9 @@ export class ChatAttachmentsContentPart extends Disposable {
 		dom.clearNode(container);
 		this.attachedContextDisposables.clear();
```

Comment:
I don't know why there's a separate disposable store on this content part, that seems redundant